src/futuresboard/app.py

```diff
@@ -34,13 +34,55 @@ def init_app(config: dict[str, Any] | None = None):
         if "DATABASE" not in config:
             config["DATABASE"] = f"{config_dir / 'futures.db'}"
 
-        if "API_BASE_URL" not in config:
-            base_url = "https://fapi.binance.com"  # production base url
-            # base_url = 'https://testnet.binancefuture.com' # testnet base url
-            config["API_BASE_URL"] = base_url
+        if "EXCHANGE" not in config:
+            base_url = "https://fapi.binance.com"
+        else:
+            if config["EXCHANGE"] == "binance":
+                base_url = "https://fapi.binance.com"
+            else:
+                base_url = "https://fapi.binance.com"  # alternatives here later
+
+        config["API_BASE_URL"] = base_url
 
         if "AUTO_SCRAPE_INTERVAL" not in config:
             config["AUTO_SCRAPE_INTERVAL"] = 5 * 60
+        else:
+            try:
+                interval = int(config["AUTO_SCRAPE_INTERVAL"])
+                if interval < 60 or interval > 3600:
+                    config["AUTO_SCRAPE_INTERVAL"] = 5 * 60
+            except:
```

Comment:
```suggestion
            except TypeError:
```
Narrow down the exception so that any other exception bubbles up.